calico-enterprise_versioned_docs/version-3.20-2/release-notes/index.mdx

```diff
@@ -249,4 +249,8 @@ May 29, 2025
 * Fixed attaching and detaching CTLB programs when using BPF dataplane.
 * Security updates.
 
+#### Known issues
+* **Important:** If you are using the Authentication resource, the Dex pod has a problem starting due to an entrypoint bug. While we are working on a patch,
+    replace the tigera-operator deployment with the following image: `quay.io/tigera/operator:v1.36.10.1`.
+
```

Comment:
```suggestion
*  When you use the `Authentication` resource, an entryway bug causes the `dex` pod to have trouble starting. As a workaround, replace the `tigera-operator` deployment with the following image: quay.io/tigera/operator:v1.36.10.1`.
This issue will be fixed in an upcoming patch release.
```
Would be best to have another line saying what happens. A second sentence like this: `User are unable to sign in to the web console.`